components/molecule/notification/src/index.js

```diff
@@ -131,9 +131,9 @@ const MoleculeNotification = ({
             </div>
           )}
         </div>
-        {buttons && (
+        {buttons.length ? (
```

Comment:
```suggestion
        {buttons?.length ? (
```

and remove the default value